scripts/fix-links.js

```
@@ -0,0 +1,213 @@
+						return `](../enumerations/${enumName})`;
+					});
+					hasChanges = true;
+				}
+			}
+		}
+
+		// ADVANCED PATTERN MATCHING for any remaining nested directory duplication
+		// Pattern: [text](./dirA/dirA/something) -> [text](./dirA/something)
+		// Pattern: [text](./dirA/dirB/dirB/something) -> [text](./dirA/dirB/something)
+		const nestedDuplicationRegex = /(\[.*?\]\([^)]*?)\/([^\/\)]+)\/\2\//g;
+		let matches = [...updatedContent.matchAll(nestedDuplicationRegex)];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'matches' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)